### PR TITLE
docs(ios): extra Pod repo update step incase 'cli init' fails

### PR DESCRIPTION
**Pull request by quarryman**
Getting started guide claims one can start new RN project with react-native-firebase integrated with next template:
`npx @react-native-community/cli init --template=@react-native-firebase/template <name>`

Latest react-native cli init will try running pod install automatically.

In case user performing install has not updated cocoapods repo index for a while, pod install will fail due to latest Firebase modules version are not found in current cocoapods repo.

@mikehardy filed an issue  to _cli_ repo https://github.com/react-native-community/cli/issues/752
until its resolved, we can update docs with extra step that solves the issue.

Solution is to run 'pod update' or 'pod install --repo-update' manually after 'cli init' fails

### Checklist

- [ ] Supports `Android`
- [ ] Supports `iOS`
- [ ] `e2e` tests added or updated in [/tests/e2e/\*](/tests/e2e)
- [x ] Updated the documentation in the [docs repo](https://github.com/invertase/react-native-firebase-docs)
  - **LINK TO DOCS PR HERE**
- [ ] Flow types updated
- [ ] Typescript types updated

### Release Plan

[CATEGORY][type] [LOCATION] - Message

---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Donate via [Open Collective](https://opencollective.com/react-native-firebase/donate)
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter
- 👉 Star this repo on GitHub ⭐️
- 👉 Contribute; see our [contributing guide](/CONTRIBUTING.md)

### CODE REVIEW

File: pages/quick-start/new-project.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ The following npx command will create a new directory on your machine with the `
 ```bash
 npx @react-native-community/cli init --template=@react-native-firebase/template <name>
 ```
+> In case `cli init` script fails while performing `pod install`, follow next step:
```
Comment (Salakar) [inline pages/quick-start/new-project.md]:
```suggestion
> If the `cli init` script fails while performing the `pod install` phase (fails with `could not find spec` or similar), try the following step:
```

**Comment by Salakar** [comment]
Thanks for the PR 🥳 

LGTM, I added a suggestion to be slightly more explicit, if it makes sense to you; we can commit it and then merge this PR.

**Comment by quarryman** [comment]
Accepted suggestion, I think it can be merged now.
Thanx for review and suggestion.

File: pages/quick-start/new-project.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ The following npx command will create a new directory on your machine with the `
 ```bash
 npx @react-native-community/cli init --template=@react-native-firebase/template <name>
 ```
+> If the `cli init` script fails while performing the `pod install` phase (fails with `could not find spec` or similar), try the following step:
```
Comment (Ehesp) [inline pages/quick-start/new-project.md]: Please remove the `>` characters, as it will cause the links to within a blockquote, with a nested code example.

**Comment by mikehardy** [comment]
@quarryman looks like this needs a small change per review above then can be merged?

File: pages/quick-start/new-project.md (hunk 2 above)
Comment (quarryman) [reply pages/quick-start/new-project.md]: Done.
However I supposed this whole block to be displayed like a notion, not just regular text.
Anyway it looks ok after removal.

**Comment by quarryman** [comment]
@mikehardy Provided changes. Can be checked and merged if looks appropriate now.
Thanx

**Comment by quarryman** [comment]
Should I squash commits btw?

**Comment by mikehardy** [comment]
I'll let @Ehesp or @Salakar decide on the merge, I was just browsing through other stuff and saw that this was in danger of going stale when it was so close otherwise and pinged you

**Comment by quarryman** [comment]
@Ehesp @Salakar guys, is any further action required to merge?

File: pages/quick-start/new-project.md [hunk 3]
```diff
@@ -14,6 +14,11 @@ The following npx command will create a new directory on your machine with the `
 ```bash
 npx @react-native-community/cli init --template=@react-native-firebase/template <name>
 ```
+If the `cli init` script fails while performing the `pod install` phase (fails with `could not find spec` or similar), try the following step:
+```bash
+$ cd ./<name>/ios
+$ pod update (or pod install --repo-update)
```
Comment (Salakar) [inline pages/quick-start/new-project.md]:
```suggestion
$ pod install --repo-update
```

**Pull request merged**